### PR TITLE
Fixed broken link

### PR DESCRIPTION
**Pull request by rahulrai-in**
The link to file was returning 404.

### CODE REVIEW

File: content/docs/1.5/scalers/prometheus.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ layout = "scaler"
 availability = "v1.0+"
 maintainer = "Community"
 description = "Scale applications based on Prometheus."
-go_file = "prometheus_scaler"
+go_file = "prometheus"
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/prometheus.md:7]: Would you mind updating our doc in v1.4 & v2.0 as well please?

Comment (rahulrai-in) [reply content/docs/1.5/scalers/prometheus.md:7]: Sure. No problem.

Comment (rahulrai-in) [reply content/docs/1.5/scalers/prometheus.md:7]: Done.

**Review by tomkerkhove — APPROVED** [review]
LGTM - Just have to fix the DCO issue and we can merge!

File: content/docs/2.0/scalers/prometheus.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ layout = "scaler"
 availability = "v1.0+"
 maintainer = "Community"
 description = "Scale applications based on Prometheus."
-go_file = "prometheus_scaler"
```
[the remark this answers is not in the document]
Comment (rahulrai-in) [reply content/docs/2.0/scalers/prometheus.md]: Addressed the review comment. Thanks.

**Review by ahmelsayed — APPROVED** [review]

**Comment by rahulrai-in** [comment]
Is it good for the merge now? Is there anything pending on it?

**Comment by tomkerkhove** [comment]
It's good to go but looks like DCO issue is blocking it.

Can you fix it by following this please? https://github.com/kedacore/keda/blob/v2/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what

**Comment by rahulrai-in** [comment]
I created a fresh PR since the master had moved quite a bit ahead of the old branch. Can you sign off this one @tomkerkhove https://github.com/kedacore/keda-docs/pull/274

**Pull request closed without merging**